src/pyOpenMS/pxds/MetaInfoInterface.pxd

```diff
@@ -26,25 +26,25 @@ cdef extern from "<OpenMS/METADATA/MetaInfoInterface.h>" namespace "OpenMS":
 
         bool operator==(MetaInfoInterface) nogil except +
         bool operator!=(MetaInfoInterface) nogil except +
-        bool isMetaEmpty() nogil except +
-        void clearMetaInfo() nogil except +
+        bool isMetaEmpty() nogil except + # wrap-doc:Returns if the MetaInfo is empty
+        void clearMetaInfo() nogil except + # wrap-doc:Removes all meta values
 
-        MetaInfoRegistry metaRegistry() nogil except +
+        MetaInfoRegistry metaRegistry() nogil except + # wrap-doc:Returns a reference to the MetaInfoRegistry
 
         # Cython has a problem with inheritance of overloaded methods, so we
         # can only declare one of the methods here (most people will want
         # to use the String-based methods).
         #
-        void getKeys(libcpp_vector[String] & keys)
+        void getKeys(libcpp_vector[String] & keys) nogil except + # wrap-doc:Fills the given vector with a list of all keys for which a value is set
```

Comment:
Was not wrapped beforehand - does this lead to an issue - autowrap error output is unusable. 

https://abibuilder.informatik.uni-tuebingen.de/jenkins/job/openms/job/PR/job/pyOMSDckr/1459/consoleFull

```
Traceback (most recent call last):
  File "create_cpp_extension.py", line 63, in <module>
    decls, instance_map = autowrap.parse(pxd_files, ".", num_processes=int(PY_NUM_THREADS))
  File "/usr/local/lib/python3.6/dist-packages/autowrap/__init__.py", line 51, in parse
    return DeclResolver.resolve_decls_from_files(files, root, num_processes)
  File "/usr/local/lib/python3.6/dist-packages/autowrap/DeclResolver.py", line 239, in resolve_decls_from_files
    return resolve_decls_from_files_single_thread(pathes, root)
  File "/usr/local/lib/python3.6/dist-packages/autowrap/DeclResolver.py", line 232, in resolve_decls_from_files_single_thread
    decls.extend(PXDParser.parse_pxd_file(full_path))
  File "/usr/local/lib/python3.6/dist-packages/autowrap/PXDParser.py", line 518, in parse_pxd_file
    result.append(handler(body, lines, path))
  File "/usr/local/lib/python3.6/dist-packages/autowrap/PXDParser.py", line 315, in parseTree
    decl = MethodOrAttributeDecl.parseTree(att, lines, pxd_path)
  File "/usr/local/lib/python3.6/dist-packages/autowrap/PXDParser.py", line 395, in parseTree
    annotations = parse_line_annotations(node, lines)
  File "/usr/local/lib/python3.6/dist-packages/autowrap/PXDParser.py", line 125, in parse_line_annotations
    assert value.strip(), "empty value for key '%s'" % key
AssertionError: empty value for key '(default'
src/pyOpenMS/CMakeFiles/pyopenms_create_cpp.dir/build.make:76: recipe for target 'src/pyOpenMS/CMakeFiles/pyopenms_create_cpp' failed
make[3]: *** [src/pyOpenMS/CMakeFiles/pyopenms_create_cpp] Error 1
CMakeFiles/Makefile2:3214: recipe for target 'src/pyOpenMS/CMakeFiles/pyopenms_create_cpp.dir/all' failed
make[2]: *** [src/pyOpenMS/CMakeFiles/pyopenms_create_cpp.dir/all] Error 2
CMakeFiles/Makefile2:3248: recipe for target 'src/pyOpenMS/CMakeFiles/pyopenms.dir/rule' failed
make[1]: *** [src/pyOpenMS/CMakeFiles/pyopenms.dir/rule] Error 2
```

Reply:
Not sure. Should I revert my changes?

Reply:
I will check it out locally and will let you know.